src/modules/shared/components/Header/Header.tsx

```diff
@@ -82,9 +86,64 @@ export const Header: React.FC = () => {
           </NavLink>
         </div>
 
-        <div className={styles.header__burger}>
-          <img src={burger} alt="burger-menu" />
-        </div>
+        <button
+          type="button"
+          className={cn([styles.header__burger_icon], {
+            [styles.header__burger__IsOpen]: IsBurgerActive,
+          })}
+          onClick={() => setIsBurgerActive(!IsBurgerActive)}
+        >
+          {IsBurgerActive
+            ? (<img src={close} alt="burger-menu" />)
+            : (<img src={burger} alt="burger-menu" />)}
```

Comment:
You can write a condition directly in `src` and you won't need to do a ternary operator